### DISCUSSION TITLE
Feature Request - Wiki preview branches & pull request

### DISCUSSION THREAD

**Discussion by pjgg** [community/community#9204] [Pull Requests]
As a developer (main GitHub target user) I would like to fork the company wiki repo, create my own branches in my repo and then make a pull request to the company repo from my custom branch. 

As a company maintainer repo, I  would like to resolve doc wiki conflicts and make suggestions for a given pull request. Finally, I would like to merge pull requests into a branch. 

The goal of the above statements is to be able to manage the documentation in an agile way but also be able to handler several future releases (the pull requests) and the production version (main branch).

**Comment by bbrewington** [comment]
bunch of people talking about these features here (this link is to an issue in a now archived repo): https://github.com/isaacs/github/issues/333
